src/components/EntityInfo/EntityInfo.tsx

```diff
@@ -13,7 +15,7 @@ export default function EntityInfo({ data }: Props): JSX.Element {
         return (
           <div className="entity-info__grid-item" key={label}>
             <h4 className="p-muted-heading">{label}</h4>
-            <p data-name={label}>{value}</p>
+            <TruncatedTooltip message={value}>{value}</TruncatedTooltip>
```

Comment:
I thought it might be better to use `TruncatedTooltip` for all fields, in case there would be other fields populated with a long string. I checked and all `value` are string, so it works well with message prop.